src/apps/frontend/app.component.tsx

```diff
@@ -17,16 +17,19 @@ export default function App(): React.ReactElement {
   }, []);
 
   return (
-    <DepsProvider deps={{
-      accessService: new AccessService(),
-    }}>
+    <DepsProvider
```

Comment:
Can we get rid of `DepsProvider` and create `AuthContextProvider` etc. Earlier we model `DepsProvider` as one central repository for dependencies but that's not react thinking.

Kindly use reference from Sunlead on this is done and refactor it please.